csharp/src/Ice/RequestFailedException.cs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+
+namespace Ice
+{
+    // Extends the partial exception classes generated by the Slice compiler.
+
+    public partial class RequestFailedException
+    {
+        protected virtual string DefaultIceMessage
+            => Facet.Length == 0 ? $"Request for operation `{Operation}' on Ice object `{Id}' failed."
+                : $"Request for operation `{Operation}' on Ice object `{Id}' with facet `{Facet}' failed.";
```

Comment:
Is the plan to change all the exception messages to start with an uppercase letter and finish with a dot? So far all our messages started with lowercase letter and didn't have a dot.

Reply:
Fixing. This PR is not about changing the error message style.